frontend-ssr/components/feedback-review/Comments.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './comments.module.scss';
+import CommentBlock from './CommentBlock';
+import { Comment } from './comment.model';
+
+interface CommentsProps {
+  comments: Comment[];
+}
+
+export default function Comments(props: CommentsProps) {
+  if (props.comments?.length === 0) return <h3>Nu exista comentarii pentru acest exercitiu</h3>;
+
+  return (
+    <div className={styles.comments}>
+      <header>
+        <h3>Comentarii</h3>
+        <button className="btn btn--blue">Aproba tot</button>
+        <button className="btn btn--danger">Respinge tot</button>
+      </header>
+      <section className="comments">
+        {props.comments?.map(comment => <CommentBlock key={comment.id} {...comment} />)}
```

Comment:
Do we need to use optional chaining here? Don't we know the comments are gonna be an Array?

Reply:
No, we didn't need that. I did some debugging when implementing and I left it here by accident